src/assets/thirdPartyServices/common/HtmlTooltip.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Tooltip from "@material-ui/core/Tooltip/index";
+import InfoIcon from "@material-ui/icons/Info";
+
+class HtmlTooltip extends React.Component {
```

Comment:
The MaterialUI tooltip has a prop called `placement` out of the box. you could use that instead of creating a wrapper.

Reply:
Yeah, Material UI provide us a standard tooltip component which has many ways to be implemented. My idea behind the wrapper is start creating a Tooltip according to Greg's design pattern to be reused by other team members, in order to don't duplicate code.

Reply:
Hi.. appreciate your efforts.. The reason for suggesting MUI-Tooltip is we're already using the same at various places in our application. so if necessary, we could customize it globally.. I will speak with the Greg/Sujith about this, so that everybody are on the same page.

Reply:
I have connected with @sujithvarma . We could use MUI-Tooltip. we will be creating a StyledTooltip with our custom styles.. Once it is done.. you could just replace the *Tooltip* with *StyledTooltip*.